kext/FileNVRAM/FileNVRAM.h

```diff
@@ -180,7 +182,7 @@ class FileNVRAM : public IODTNVRAM
     OSDictionary * mNvramMissDict;
     IOCommandGate* mCommandGate;
     OSString*      mFilePath;
-    IOTimerEventSource* mTimer;
+    IONotifier* disk_notifier_start_;
```

Comment:
Good catch. 

Yes, I'f the existing code works with that fix, I'd try just making that (minimal) change.